src/dashboard/src/media/js/archival_storage/archival_storage_search.js

```diff
@@ -101,33 +100,314 @@ $(document).ready(function() {
+        {sTitle: gettext('File count'), mData: 'file_count', defaultContent: ''},
+        {sTitle: gettext('Accession numbers'), mData: 'accessionids', mRender: render_aip_accession_ids},
+        {sTitle: gettext('Created'), mData: 'created', mRender: render_aip_created_date },
+        {sTitle: gettext('Status'), mData: 'status', mRender: render_status },
+        {sTitle: gettext('Encrypted'), mData: 'encrypted', mRender: render_aip_encrypted },
+        {sTitle: gettext('Actions'), mData: 'uuid', mRender: render_aip_actions }
+      ];
+    }
+
+    return $('#archival-storage-entries').dataTable({
+        'dom': 'rtiBp',
+        'stateSave': true,
+        'stateDuration': 60 * 60 * 24 * 365 * 10, // set state duration to 10 years
+        // Only save column visibility
+        'stateSaveParams': function(settings, data) {
+            delete data.search;
+            delete data.start;
+            delete data.order;
+            delete data.length;
+            for ( var i=0 ; i< data.columns.length ; i++ ) {
+              delete data.columns[i].search;
+            }
+        },
+        'stateSaveCallback': function(settings, data) {
+            $.ajax({
+                url: '/archival-storage/save_state/' + get_state_url_params(),
+                data: JSON.stringify(data),
+                dataType: 'json',
+                type: 'POST',
+                success: function() {}
+            });
+        },
+        'stateLoadCallback': function (settings, callback) {
+            $.ajax({
+                url: '/archival-storage/load_state/' + get_state_url_params(),
+                dataType: 'json',
+                success: function (json) {
+                  callback(json);
+                },
+                // Handle case where there is no saved state
+                error: function () {
+                  callback(null);
+                }
+            });
+        },
+        'columnDefs': [
+          {
+            'targets': get_default_hidden_column_indices(),
+            'visible': false
+          },
+          {
+            'targets': get_unorderable_column_indices(),
+            'orderable': false
+          }
+        ],
+        'buttons': [
+          {
+              'extend': 'colvis',
+              'text': '<i class="fa fa-cog" aria-hidden="true"></i> ' + gettext('Select columns')
+          }
+        ],
+      'language': {
+        'sEmptyTable':            pgettext('DataTable - sEmptyTable',         'No data available in table'),
+        'sInfo':                  pgettext('DataTable - sInfo',               'Showing _START_ to _END_ of _TOTAL_ entries'),
+        'sInfoEmpty':             pgettext('DataTable - sInfoEmpty',          'Showing 0 to 0 of 0 entries'),
+        'sInfoFiltered':          pgettext('DataTable - sInfoFiltered',       '(filtered from _MAX_ total entries)'),
+        'sInfoPostFix':           '', // pgettext('DataTable - sInfoPostFix',        ''),
```

Comment:
Do you have more context for this line? Should the part following `//` be removed?

Reply:
Yes, this should be (and now has been) removed. It allows you to add a message to the table such as "All records shown are derived from real information." ((from legacy DataTables docs)[https://legacy.datatables.net/ref]), but we don't use this feature.